setup.py

```diff
@@ -24,25 +24,24 @@
       install_requires=[
           "requests>=2.31.0,<3",
           "pyjwt>=2,<3",
-          "arrow>=0.15.4,<2",
-          "gemd>=1.15.0,<2",
-          "boto3>=1.17.93,<2",
-          "botocore>=1.20.95,<2",
+          "arrow>=1.3.0,<2",
```

Comment:
I'd grant a little more grace on minimum version here, but requiring >= v1 would certainly make sense.

Reply:
I don't feel super strongly on these lower bounds, so sure, I'll drop it to 1.0.